pkg/minikube/node/config.go

```diff
@@ -66,7 +66,9 @@ func configureRuntimes(runner cruntime.CommandRunner, drvName string, k8s config
 	if driver.BareMetal(drvName) {
 		disableOthers = false
 	}
-	if !driver.IsKIC(drvName) {
+
+	// Preload is overly invasive for bare metal, and caching is not meaningful.
+	if driver.IsVM(drvName) {
```

Comment:
how about Kic?

Reply:
maybe use, if !driver.BareMetal(drvName)  instead

Reply:
The previous if statement excluded KIC, IsVM preserves that. Was Preload previously broken in KIC, or am I missing something?

Reply:
ah my bad, for kic we extract it differently than VM.
it would worth adding a comment with reference where we do it for kic. (in kic driver create)